### PR TITLE
removed prototype data from page templates to resolve bugs

### PR DESCRIPTION
**Pull request by kave**
@richaagarwal I went and lifted your changes specifically to the page templates from your https://github.com/cfpb/cfgov-refresh/pull/1393 into a separate PR. due to some bugs creeping in with the sidebar-contact-molecule.

@kurtw 

### Testing
- Create individual Pages in wagtail and verify they don't break
 - landing page
 - sublanding page
 - browse page
 - browse filterable page
 - document detail page
 - learn page

### CODE REVIEW

File: cfgov/jinja2/v1/_includes/organisms/sidebar-contact-info.html [hunk 1]
```diff
@@ -49,38 +49,40 @@ <h2 class="header-slug">
...
-    <div class="o-sidebar-contact-info_column">
-        {% if value.contact.contact_info %}
-            {% for block in value.contact.contact_info %}
-                {% if not block.block_type == 'address' %}
-                    {{ render_stream_child(block) }}
-                {% endif %}
-            {% endfor %}
-        {% endif %}
-    </div>
+    {% if value %}
```
Comment (kurtw) [inline cfgov/jinja2/v1/_includes/organisms/sidebar-contact-info.html]: This conditional should be removed.

File: cfgov/jinja2/v1/document-detail/index.html [hunk 2]
```diff
@@ -106,66 +33,27 @@
...
-                        {{ render_stream_child(block) }}
-                    </h3>
-                {% elif 'slug' in block.block_type %}
-                    <h2 class="header-slug">
+{% for block in page.sidefoot %}
+    <div class="block block__flush-top">
+        {% if 'related_posts' in block.block_type %}
+            {{ related_posts.render(block) }}
+        {% else %}
+            {% if 'heading' in block.block_type %}
```
Comment (kurtw) [inline cfgov/jinja2/v1/document-detail/index.html]: Could combine the conditionals

**Comment by richaagarwal** [comment]
:+1: (My PR is still a WIP)

File: cfgov/jinja2/v1/landing-page/index.html [hunk 3]
```diff
@@ -94,32 +46,27 @@
...
-                        </h2>
-                    {% elif 'paragraph' in block.block_type %}
-                        {{ render_stream_child(block) }}
-                    {% else %}
+    {% for block in page.sidefoot %}
+        <div class="block block__flush-top">
+            {% if 'related_posts' in block.block_type %}
+                {{ related_posts.render(block) }}
+            {% else %}
+                {% if 'heading' in block.block_type %}
```
Comment (kurtw) [inline cfgov/jinja2/v1/landing-page/index.html]: Combine conditionals.

File: cfgov/jinja2/v1/learn-page/index.html [hunk 4]
```diff
@@ -228,49 +31,27 @@
...
-                            {{ render_stream_child(block) }}
-                        </span>
-                    </h2>
-                {% elif 'paragraph' in block.block_type %}
+    {% for block in page.sidefoot %}
+    <div class="block block__flush-top">
+        {% if 'related_posts' in block.block_type %}
+            {{ related_posts.render(block) }}
+        {% else %}
+            {% if 'heading' in block.block_type %}
```
Comment (kurtw) [inline cfgov/jinja2/v1/learn-page/index.html]: Possibly combine conditionals.

File: cfgov/jinja2/v1/learn-page/index.html [hunk 5]
```diff
@@ -1,42 +1,7 @@
...
-    %}
-    {% set page = page or global_dict.page %}
-    {% set prototype = global_dict.prototype or false %}
-    {% set breadcrumb_items = [('#', '#', 'Sample Breadcrumb')] %}
-{% endif %}
 
-{% import 'atoms/large-checkbox.html' as large_checkbox %}
-{% import 'atoms/large-radio.html' as large_radio %}
-
-{% import 'molecules/related-posts.html' as related_posts with context %}
```
Comment (kurtw) [inline cfgov/jinja2/v1/learn-page/index.html]: This needs to stay in.

**Comment by kurtw** [comment]
Other than the conflicts, the conditional statements (which could stay in if we're just removing them anyway), and adding the `related_posts` back into the learn_page/index.html it's good to go.

File: cfgov/jinja2/v1/learn-page/index.html (hunk 4 above)
Comment (kave) [inline cfgov/jinja2/v1/learn-page/index.html]: done!

File: cfgov/jinja2/v1/learn-page/index.html (hunk 5 above)
Comment (kave) [inline cfgov/jinja2/v1/learn-page/index.html]: put back in

**Comment by kave** [comment]
@kurtw i left the atoms in the sidefoot section for now. i think it mores more sense to remove it in your PR removing the models

**Comment by kurtw** [comment]
:+1:

**Pull request merged**